test/e2e/e2e_test.go

```diff
@@ -464,6 +472,124 @@ var _ = ginkgo.Describe("Kueue", func() {
 		})
 	})
 
+	ginkgo.When("QueueingPolicy", func() {
+		var (
+			onDemandRF   *kueue.ResourceFlavor
+			spotRF       *kueue.ResourceFlavor
+			localQueue   *kueue.LocalQueue
+			clusterQueue *kueue.ClusterQueue
+		)
+		ginkgo.BeforeEach(func() {
+			onDemandRF = testing.MakeResourceFlavor("on-demand").
+				Label("instance-type", "on-demand").Obj()
+			gomega.Expect(k8sClient.Create(ctx, onDemandRF)).Should(gomega.Succeed())
+			spotRF = testing.MakeResourceFlavor("spot").
+				Label("instance-type", "spot").Obj()
```

Comment:
I don't think this flavor is relevant for the test. Remove it to keep the test simple.

Reply:
Okok I will use "default-flavor" to keep it simple instead